### PR TITLE
chore!: update testnet URLs

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: https://github.com/FuelLabs/fuels-ts/issues/2111

## Breaking Changes:

- Constant `FUEL_BETA_5_NETWORK_URL` was [renamed](https://github.com/FuelLabs/fuels-ts/pull/2272/commits/5fb020877e48c270bae93f6ab2279d126fca9776#diff-3ea5b95eb0b1d85ccfe3f01526579bd5a380f56010d977df8938c268d261f768L10-L12) to `FUEL_TESTNET_NETWORK_URL`

### CODE REVIEW

**Comment by Torres-ssf** [comment]
Blocked until the Block explorer is updated to `fuel-core` ~ `0.26`

**Comment by Torres-ssf** [comment]
> Blocked until the Block explorer is updated to `fuel-core` ~ `0.26`

After speaking with @LuizAsFight I could validate that the block explorer URL will not change after the upgrade, therefore, this PR is no longer blocked.

**Comment by fuel-service-user** [comment]
✨ A PR has been created under the `rc-2272` tag on `fuels-wallet` repo.
https://github.com/FuelLabs/fuels-wallet/pull/1306

**Review by nedsalk — APPROVED** [review]

**Review by maschad — APPROVED** [review]

File: packages/fuel-gauge/src/e2e-script.test.ts [hunk 1]
```diff
@@ -31,8 +31,8 @@ describe('Live Script Test', () => {
       console.error((e as Error).message);
       console.warn(`
         not enough coins to fit the target?
-        - add assets: https://faucet-beta-5.fuel.network/
-        - check balance: https://fuellabs.github.io/block-explorer-v2/beta-5/#/address/${address}
+        - add assets: https://faucet-devnet.fuel.network/
+        - check balance: https://app.fuel.network/account/${address}/assets/
```
Comment (arboleya) [inline packages/fuel-gauge/src/e2e-script.test.ts:35]: Will we use the same URL for the explorer?

**Review by arboleya — APPROVED** [review]

File: packages/fuel-gauge/src/e2e-script.test.ts (hunk 1 above)
Comment (Torres-ssf) [reply packages/fuel-gauge/src/e2e-script.test.ts:35]: @arboleya according to @LuizAsFight the URL will remain the same.

**Review by danielbate — APPROVED** [review]

**Pull request merged**